x-pack/test/functional/apps/infra/metrics_source_configuration.ts

```diff
@@ -49,12 +50,31 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         await metricIndicesInput.clearValueWithKeyboard({ charByChar: true });
         await metricIndicesInput.type('does-not-exist-*');
 
-        await infraSourceConfigurationForm.saveConfiguration();
+        await infraSourceConfigurationForm.saveInfraSettings();
 
         await pageObjects.infraHome.waitForLoading();
         await pageObjects.infraHome.getInfraMissingMetricsIndicesCallout();
       });
 
+      it('can clear the input and reset to previous values without saving', async () => {
```

Comment:
I added this check to verify if the discard action is also working ( resetting the previous value )